lstchain/scripts/lstchain_create_dl3_index_files.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+"""
+A separate script to index the HDU tables and Observation tables of the DL3 files
+in a particular directory.
+The number of DL3 fits files to be indexed from the directory can be selected as
+per a sorted order.
+
+Usage:
+$> python lstchain_create_dl3_index_files.py
+--input-dl3-dir ./DL3/
+--num-files n
+"""
+
+import os
+from pathlib import Path
+import logging
+import sys
+import argparse
+import numpy as np
+
+from lstchain.irf import create_obs_hdu_index
+
+log = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(description="Creating DL3 index files")
+
+# Required arguments
+parser.add_argument('--input-dl3-dir', '-d', type=Path,
+                    dest='input_dl3_dir',
+                    help='path to DL3 files',
+                    default=None, required=True
+                    )
+
+parser.add_argument('--num-files', '-n', type=int,
```

Comment:
This is supposed to take the n files from the folder and process them?
A more general way, that only needs one argument,  would be to provide an expression.
So instead of `path/to/files` you could give something like `path/to/files/run00[3-4]*`
This could be fed into pathlibs glob or the python standard library glob to create the filelist.

Reply:
Thank you for this suggestion! I have implemented it and now the script is much crisper!